Assets/Scripts/Content/QuestData.cs

```diff
@@ -1179,8 +1179,8 @@ override public string ToString()
     public class Puzzle : Event
     {
         new public static string type = "Puzzle";
-        public string puzzleClass = "slide";
-        public string skill = "{observation}";
+        public StringKey puzzleClass = new StringKey("val", "PUZZLE_SLIDE_CLASS");
```

Comment:
Ok I will use the uppercase magic string name in order to be easily imported from an existing quest. I will modify the puzzle load from existing quest in order to replace it.